pipelines/DRP.yaml

```diff
@@ -199,3 +202,38 @@ subsets:
 
       This subset is considered a workaround for missing middleware and task
       functionality.  It may be removed in the future.
+  step4:
+    subset:
+    - forcedPhotCcd
+    - forcedPhotDiffim
+    - imageDifference
+    - transformDiaSourceCat
+    description: >
+      Tasks that can be run together, but only after the 'step1', 'step2'
+      and 'step3' subsets
+
+      These detector-level tasks should not be run with
+      'tract' or 'patch' as part of the data ID expression if all
+      reference catalogs or diffIm templates that cover these
+      detector-level quanta are desired.
+  step5:
+    subset:
+    - consolidateDiaSourceTable
+    - drpAssociation
+    - drpDiaCalculation
+    description: >
+        Tasks that can be run together, but only after the 'step1', 'step2',
+        'step3', and 'step4' subsets
+
+        These DIA aggregation steps include both visit-level aggregation and
+        tract-level aggregation drpAssociation and drpDiaCalculation.
+
+        If run with tract or patch in the data query, drpAssociation and
+        drpDiaCalculation will be complete, but the outputs of
+        consolidateDiaSourceTable not complete visits.
```

Comment:
Like obs_lsst, I don't understand this sentence, it's missing a verb. Also, should this be run with a tract or patch or without a tract or patch? That's not clear.